Core/EasylistStore.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  EasylistStore.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+import Foundation
+
+class EasylistStore {
+
+    static let shared = EasylistStore()
+
+    enum Easylist: String {
+
+        case easylist
+        case easylistPrivacy
+
+    }
+
+    var hasData: Bool {
+        return easylist != "" && easylistPrivacy != ""
+    }
+
+    private(set) var easylist: String = ""
+    private(set) var easylistPrivacy: String = ""
+
+    private init() {
+        easylist = (try? Data(contentsOf: persistenceLocation(type: .easylist)).base64EncodedString()) ?? ""
+        easylistPrivacy = (try? Data(contentsOf: persistenceLocation(type: .easylistPrivacy)).base64EncodedString()) ?? ""
+    }
+
+    func persistEasylist(data: Data) {
+        easylist = data.base64EncodedString()
+        try! data.write(to: persistenceLocation(type: .easylist), options: .atomic)
```

Comment:
Under what conditions does this throw? Are we sure we want to crash the app?

Reply:
Changed to `try?`